tests/cxx/namespace_use.cc

```diff
@@ -0,0 +1,45 @@
+//===--- namespace_use.cc - test input file for iwyu ----------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// IWYU_ARGS: -I .
+
+// Tests that using-directives i.e. `using namespace`, get appropriate
+// include suggestions.
```

Comment:
It looks like this can be rewrapped to fit `include` on the first line.

Also I tend to avoid backticks in code, but maybe only because it feels like an exotic character on my Swedish keyboard :)